app/src/main/res/values/dimens.xml

```diff
@@ -3,4 +3,14 @@
     <dimen name="square_image_dimension">64dp</dimen>
     <dimen name="card_radius">10dp</dimen>
     <dimen name="card_elevation">4dp</dimen>
+    <dimen name="preview_badge_padding">135dp</dimen>
+    <dimen name="spinner_padding">8dp</dimen>
+    <dimen name="text_size">16sp</dimen>
```

Comment:
Too generic

Reply:
Since this is the most commonly used textSize in whole app that's why I have kept this generic

Reply:
Still, make it specific, text_size_normal